confluence-xml/src/main/java/org/xwiki/contrib/confluence/filter/input/ConfluenceXMLPackage.java

```diff
@@ -89,9 +89,15 @@ public class ConfluenceXMLPackage implements AutoCloseable
      */
     public static final String FILE_DESCRIPTOR = "exportDescriptor.properties";
 
+    /**
+     * The property key to access to the CSV that contains task related informations.
+     */
+    public static final String FILE_TASKS = "AO_BAF3AA_AOINLINE_TASK.csv";
```

Comment:
That name sounds quite a lot like a name generated from various criteria, more that something that always have this specific name.